examples/basic-create-react-app-graphql/src/services/neo4jDesktopService.js

```diff
@@ -0,0 +1,81 @@
+/* global neo4jDesktopApi */
+
+import {split} from 'apollo-link';
+import {createHttpLink} from "apollo-link-http";
+import {WebSocketLink} from "apollo-link-ws";
+import {setContext} from "apollo-link-context";
+import ApolloClient from "apollo-client";
+import {InMemoryCache} from "apollo-cache-inmemory";
+import {getMainDefinition} from 'apollo-utilities';
+
+import getWorkspaceQuery from './getWorkspaceQuery';
+import onWorkspaceChangeSub from './onWorkspaceChangeSub';
+
+export default (store) => {
+    store.dispatch({
+        type: 'API_VERSION',
+        version: neo4jDesktopApi.version
```

Comment:
This is coupling to the injected API. I don't think we should do that. Maybe version should be a URL param as well?

Reply:
It would lead to changes in the desktop. Might be a good thing to do after the release.